src/ParallelAlgorithms/Interpolation/InterpolationTargetDetail.hpp

```diff
@@ -668,7 +668,7 @@ void compute_dest_vars_from_source_vars(
       InterpolationTargetTag::compute_vars_to_interpolate::apply(
           dest_vars, source_vars, mesh, jac_grid_to_inertial,
           invjac_grid_to_inertial, jac_logical_to_grid, invjac_logical_to_grid,
-          inertial_mesh_velocity);
+          inertial_mesh_velocity, tnsr::I<DataVector, 3, Frame::Grid>{});
```

Comment:
Please document this call signature. I see there's a protocol, so document it there.

Reply:
The protocol docs just point to the example, so I updated the example (which was actually incorrect but we never checked that the examples conformed to the protocol?? so I added some asserts as well)

Reply:
I think the Example.hpp isn't actually compiled, so the asserts don't do anything.

Reply:
Ok added cmake stuff so it's compiled